klog.go

```diff
@@ -650,44 +655,68 @@ func (buf *buffer) someDigits(i, d int) int {
 	return copy(buf.tmp[i:], buf.tmp[j:])
 }
 
-func (l *loggingT) println(s severity, args ...interface{}) {
+func (l *loggingT) println(s severity, logr logr.InfoLogger, args ...interface{}) {
```

Comment:
If you store logr in the loggingT already, why pass it to every method?

Reply:
We pass it around because this `logr` is an *InfoLogger*, and so it can be levelled by calls to `V()` without println needing to be aware of levelling